src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationRefresher.cs

```diff
@@ -9,18 +10,36 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
     internal class AzureAppConfigurationRefresher : IConfigurationRefresher
     {
         private AzureAppConfigurationProvider _provider = null;
+        private ILoggerFactory _loggerFactory;
```

Comment:
Can we always get/set the loggerFactory from/to the provider instead of keeping a local copy? I'm losing track of which one is the source of truth.

Reply:
Good idea, I'll update that